Phonebook.Frontend/src/app/pages/dashboard/dashboard.component.html

```diff
@@ -1,133 +1,147 @@
-
-              "
-            >
-              Custom Order</mat-option
-            >
-            <mat-option
-              value="asc"
-              i18n="DashboardComponent|The alphabetical asc order of the favorite cards@@DashboardComponentOrderAsc"
-            >
-              Alphabetical asc</mat-option
-            >
-            <mat-option
-              value="desc"
-              i18n="DashboardComponent|The alphabetical desc order of the favorite cards@@DashboardComponentOrderDesc"
-            >
-              Alphabetical desc</mat-option
-            >
-          </mat-select>
-        </mat-form-field>
-      </div>
-      <!-- This is a workaround for https://github.com/angular/material2/issues/13372 -->
-      <div cdkDropListGroup class="pb-bookmarks-list pb-flex-row">
-        <div
-          *ngFor="let person of bookmarkedPersons; index as i"
-          cdkDropList
-          [cdkDropListData]="i"
-          cdkDropListOrientation="horizontal"
-          class="card-container"
-        >
-          <app-user-small-card
-            [person]="person"
-            [actionButtonIcon]="'close'"
-            [actionButtonClasses]="'white'"
-            (actionButtonClicked)="removeFromBookmarkedPersons(person)"
-            class="pb-small-card pb-card"
-            cdkDrag
-            (cdkDragEntered)="entered($event)"
-            (cdkDragEnded)="ended($event)"
-            [cdkDragData]="i"
-            [cdkDragDisabled]="favoriteSort != ''"
-          ></app-user-small-card>
-        </div>
-      </div>
-      <div
-        *ngIf="bookmarkedPersons.length === 0"
-        i18n="DashboardComponent|Message displayed if no people are bookmarked@@DashboardComponentDescriptionBookmark"
-      >
-        You haven't bookmarked anybody yet, look for the button
-        <button mat-icon-button>
-          <mat-icon>bookmark_border</mat-icon>
-        </button>
-        on a workmates page.
-      </div>
-    </div>
-    <div class="toggle-drawer">
-      <button
-        mat-icon-button
-        (click)="drawer.toggle()"
-        i18n-matTooltip="
-           DashboardComponent|Tooltip for expanding recently viewed people@@DashboardComponentButtonExpandTooltip
-        "
-        i18n-aria-label="
-           DashboardComponent|Aria Label for expanding recently viewed people@@DashboardComponentButtonExpandArialLabel
-        "
-        matTooltip="Expand or Collapse recently viewed people"
-        aria-label="Button expanding or collapsing the recently viewed people section"
-      >
-        <span>
-          <mat-icon>{{ drawer.opened ? 'keyboard_arrow_up' : 'keyboard_arrow_down' }} </mat-icon>
-          <span i18n="DashboardComponent|Title of the recent people section@@DashboardComponentTitleRecent"
-            >Recent People</span
-          >
-        </span>
-      </button>
-    </div>
-  </mat-drawer-content>
-</mat-drawer-container>
+<div class="dashboard">

```

Comment:
falsch gemerged, hier müsste der Drawer sein. 
Kurzer Tipp am Rande: bevor du anfängst hol dir jedes mal den aktuellsten Stand vom Master, damit hältst du die Merge Konflikte gering.

Reply:
Was kann ich da jetzt machen?

Reply:
den vorherigen stand der Datei wiederherstellen und dann deine Änderungen wieder darauf anwenden

Reply:
https://stackoverflow.com/questions/5350278/git-undo-auto-merging-on-a-specific-file-only-not-the-whole-branch

Reply:
Kann ich irgendwie zum Commit vor dem merge zurück?

Reply:
ja mit dem link von oben sollte das funktionieren